src/interactions/abstractInteraction.ts

```diff
@@ -17,6 +17,34 @@ export module Interaction {
       this._componentToListenTo = component;
       this._hitBox = hitBox;
     }
+
+    /**
+     * Translates an <svg>-coordinate-space point to Component-space coordinates.
+     *
+     * @param {Point} p A Point in <svg>-space coordinates.
+     *
+     * @return {Point} The same location in Component-space coordinates.
+     */
+    protected _translateToComponentSpace(p: Point): Point {
+      var origin = this._componentToListenTo.originToSVG();
+      return {
+        x: p.x - origin.x,
+        y: p.y - origin.y
+      };
+    }
+
+    /**
+     * Checks whether a Component-coordinate-space Point is inside the Component.
+     *
+     * @param {Point} p A Point in Coordinate-space coordinates.
+     *
+     * @return {boolean} Whether or not the point is inside the Component.
+     */
+    protected _isInsideComponent(p: Point) {
```

Comment:
This checks based on the Point being in Coordinate-space.  Would it be handy to make a similar function in svg coordinate space?